README.md

```diff
@@ -205,6 +205,32 @@ Once traces have been generated, you can view them using the
 Copy `OpenCL.dll` into a system location or alongside the application executable
 you want to run with clvk.
 
+### Raspberry Pi
+
+Make sure you have an up-to-date Mesa installed on your system.
+At the time of writing (May 2023) RaspberryPi OS (Debian 11) did not, but Ubuntu
+23.04 does have a compatible vulkan driver.
+
+Install the prerequisites:
+```
+$ sudo apt install mesa-vulkan-drivers vulkan-tools libvulkan-dev git cmake clang clinfo
+```
+
+Check if your vulkan implementation has the `VK_KHR_storage_buffer_storage_class`
+extension. Note that it's not enough if this extension shows up for llvmpipe.
+You need v3dv to have this extension.  If it is not there, your Mesa is too old.
+
+Before you run `./external/clspv/utils/fetch_sources.py` you need to edit its
+first line. Change `python` into `python3` so that the interpreter will be found.
+
+Building will take a very long time on a rPi4.
```

Comment:
Did you time it? I'd be curious to know and an approximate build duration would be helpful to record here to set expectations.